main.js

```diff
@@ -25,6 +25,14 @@ autoUpdater.logger.transports.file.level = 'info';
  */
 require('electron-debug')({ showDevTools: false });
 
+/**
+ * Enable automatic reloads in development mode.
+ */
+
```

Comment:
style: spurious empty line :stuck_out_tongue_closed_eyes:

Reply:
I taught you well :-)